src/Symfony/Component/HttpKernel/Tests/UriSignerTest.php

```diff
@@ -15,6 +15,21 @@
 
 class UriSignerTest extends \PHPUnit_Framework_TestCase
 {
+    /** @var string backup original value for ini setting */
+    protected $argSeparatorOutputBackup;
```

Comment:
You don't need to backup it in whole test case when you change it's value only in one test.